src/main/java/org/reactome/release/download_directory/Biopax.java

```diff
@@ -1,4 +1,4 @@
-package org.reactome.release.downloadDirectory;
+package org.reactome.release.download_directory;
```

Comment:
underscores in package names is not conventional to Java package naming standards. It should simply be `org.reactome.release.downloaddirectory`.

Reply:
My first thought was to use a hyphen to match our project names, but a hyphen is an illegal character for a package name.  When I did a search to find out what to do for names with hyphens, the suggestion on StackOverflow was underscores according to the Java Language Specification: https://stackoverflow.com/a/3960773

Reply:
It says
>If the _domain_ name contains a hyphen...
So if our domain name was reactome-pathways.org then the package would be org.reactome_pathways...

I don't think the suggestion applies to the parts of the package beyond to domain part.

Reply:
It is just a suggestion of the Java Language Spec, though.  I can just remove the hyphen and have it be `downloaddirectory`.

Reply:
Ah okay, I didn't notice the domain name part.  I will change it to just remove the hyphen and let it be `downloaddirectory`.

Reply:
Also this https://stackoverflow.com/a/3179221 and this https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html suggest the underscore only is applicable for the domain name part of the package. I also don't think I've _ever_  seen a package with an underscore in the name.

Reply:
Okay, changed to `downloaddirectory`: https://github.com/reactome/release-download-directory/pull/1/commits/333c293219ae8a4a8d0baf7bb496fe8552965509